core/mbed/source/interrupts.cpp

```diff
@@ -26,6 +26,11 @@ void vIRQ_SetVectorX(uint32_t irqn, uint32_t vector, uint32_t flag)
     }
 }
 
+void vIRQ_SetVector(uint32_t irqn, uint32_t vector)
```

Comment:
Yes that is an option! `vIRQ_SetVector` keeps backward-compatibility with the corresponding NVIC API, meaning that it uses the same format `(irqn, vector)`. `vIRQ_SetVectorX` is somehow an extension, since it includes an additional flag -- although currently unused.

@meriac what do you think? Are we willing to keep the flag of `vIRQ_SetVectorX` or do we prefer sticking with the classic API?

For reference:
```C
void vIRQ_SetVector(uint32_t irqn, uint32_t vector);
void vIRQ_SetVectorX(uint32_t irqn, uint32_t vector, uint32_t flag);
```